README.md

```diff
@@ -27,9 +27,16 @@ $ curl https://raw.githubusercontent.com/kubevirt/hyperconverged-cluster-operato
 ```
 
 ## Installing HCO using kustomize (Openshift OLM Only)
```

Comment:
pardon my ignorance, but I don't manager to recall the difference between this paragraph and the one titled "Installing HCO Community Operator (OpenShift Only)".

Reply:
The _Installing HCO Community Operator (OpenShift Only)_ section is just manifests for namespace and subscription. We should remove it because it doesn't work:
1. The HCO operator has been removed from OperatorHub.
2. The mentioned CatalogSource _installed-community-kubevirt-hyperconverged_ does not exist by default.
3. OperatorGroup is missing.
This section is a relic from the past.
@tiraboschi do you agree?

Reply:
the section in question has been removed.

Reply:
yes, I agree we should remove this section until we will publish it again on community operators